tasks/go.py

```diff
@@ -90,6 +90,16 @@ def golangci_lint(
     raise Exit(code=1)
 
 
+@task
+def internal_deps_checker(ctx, formatFile=False):
+    """
+    Check that every required internal dependencies are correctly replaced
+    """
+    extra_params = "--formatFile true" if formatFile else ""
+    for mod in DEFAULT_MODULES.values():
```

Comment:
one caveat of this is it expects the list of DEFAULT_MODULES to be accurate. I don't know if we could compute it dynamically (I think I already asked the question and the answer is no :) )

Reply:
All of our invoke task rely on this map so for now I'll rely on it as well. I'll look into improving it if it proves to be necessary to compute it.